app/assets/stylesheets/fuel/_post-list.scss

```diff
@@ -18,11 +19,12 @@ $post-date-color:   $grey-dark;
   -webkit-overflow-scrolling: touch;
 }
 
-.post-item{
+.resource-item{
   @include rem(padding, $post-item-padding);
   border-bottom: $post-item-border;
+  margin-top: 0px;
 
-  .post-item--date{
+  .post-item--date, .author-item--title{
```

Comment:
Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space